packages/sidecar/src/formatters/logfmt/traces.ts

```diff
@@ -1,58 +1,61 @@
 import logfmt from "logfmt";
 import type { SentryTransactionEvent } from "../../parser/index.js";
+import { formatTimestamp, mapFields, mapSdkFields, mapTags } from "../utils.js";
 
 export function formatTrace(event: SentryTransactionEvent): string {
   const data: Record<string, any> = {
     type: "trace",
     event_id: event.event_id,
+    timestamp: formatTimestamp(event.timestamp),
   };
 
-  if (Number.isFinite(event.timestamp)) {
-    data.timestamp = new Date(event.timestamp * 1000).toISOString();
-  }
-
   const trace = event.contexts?.trace;
   if (trace?.trace_id) {
-    data.trace_id = trace.trace_id;
-    if (trace.span_id) data.span_id = trace.span_id;
-    if (trace.parent_span_id) data.parent_span_id = trace.parent_span_id;
-    if (trace.op) data.op = trace.op;
-    if (trace.status) data.status = trace.status;
-    if (trace.description) data.description = trace.description;
+    mapFields(trace, data, {
+      trace_id: "trace_id",
+      span_id: "span_id",
+      parent_span_id: "parent_span_id",
+      op: "op",
+      status: "status",
+      description: "description",
+    });
   }
 
-  if (event.transaction) data.transaction = event.transaction;
+  const { timestamp, start_timestamp } = event;
+  if (typeof timestamp === "number" && typeof start_timestamp === "number") {
+    data.duration_ms = Math.round((timestamp - start_timestamp) * 1000);
+  }
```

Comment:
You should be able to standardize this one too with `getSpanDuration` or something along those lines.

Reply:
ok done!